ui/test/message_popup/hint_card_test.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {mount, shallow} from 'enzyme';
+import {expect} from 'chai';
+import sinon from 'sinon';
+
+import RaisedButton from 'material-ui/RaisedButton';
+import HintCard from '../src/message_popup/hint_card.jsx';
```

Comment:
Yeah sorry, moving these files was a last-minute refactoring and I didn't check it before.  Thanks for the helpful review!